src/gpg/open.py

```diff
@@ -0,0 +1,60 @@
+import gpg
+import os
+import sys
+import time
+import textwrap
+
+
+def print_signatures(verify_result):
+    c = gpg.Context()
+    for signature in verify_result.signatures:
+        user = c.get_key(signature.fpr).uids[0].uid
+        fpr = signature.fpr
+        signed_time = time.ctime(signature.timestamp)
```

Comment:
Additional variables(fpr, signed_time) seems unnecessary to me only for the purpose of formatting, don't you think?
In GPGME docs, we have: 
```
format(c.get_key(sign.fpr).uids[0].uid,
       sign.fpr, time.ctime(sign.timestamp)))

Reply:
Ok will update in another commit. I took this from verify.py. 
In GPGME docs, these were being used with {0}, {1}, {2} so they were not looking fine to me as they didn't signify their intent. So, In verify.py I first used these variables with {0}, {1}, {2}. Later I updated formatting with named variables in so this is a remains from there.  Will remove these and use something like
```
.
.
.
               '''.format(user=c.get_key(signature.fpr).uids[0].uid,
                             fingerprint=signature.fpr,
                             time=time.ctime(signature.timestamp))
```
This will signify the intent and be concise as well.